### ISSUE TITLE
finetuning confusion

### ISSUE THREAD

**Issue by lolongcovas** [rbgirshick/rcnn#38]
Hi,
I am a bit confusing about finetuning step. In your paper there are different evaluation results for finetuning of different full-connected layers (fc5, fc6 and fc7). However, in the prototext of this repository, you just changed the name of last layer (softmax) from 'fc80 to 'fc8_pascal' and its parameters, especially, the output parameters: from 1000 to 21 classes. So, if I am not wrong, the finetuning is performed in this layer (fc8) rather than fc5, fc6, and fc7, right? 

Best
